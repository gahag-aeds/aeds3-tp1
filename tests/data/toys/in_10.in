10
0.0000 0.82685 0.3936 0.79445 0.48001 0.79275 0.79858 0.96967 0.76334 0.04773 
0.17315 0.0000 0.66859 0.97693 0.19402 0.832 0.57529 0.97671 0.2628 0.77593 
0.6064 0.33141 0.0000 0.66093 0.60159 0.61313 0.76384 0.06894 0.99 0.97085 
0.20555 0.02307 0.33907 0.0000 0.73995 0.96907 0.87326 0.55761 0.4678 0.15882 
0.51999 0.80598 0.39841 0.26005 0.0000 0.4744 0.30833 0.01072 0.02819 0.70784 
0.20725 0.168 0.38687 0.03093 0.5256 0.0000 0.40828 0.26681 0.1914 0.01377 
0.20142 0.42471 0.23616 0.12674 0.69167 0.59172 0.0000 0.69548 0.00055 0.63982 
0.03033 0.02329 0.93106 0.44239 0.98928 0.73319 0.30452 0.0000 0.24373 0.87854 
0.23666 0.7372 0.01 0.5322 0.97181 0.8086 0.99945 0.75627 0.0000 0.44558 
0.95227 0.22407 0.02915 0.84118 0.29216 0.98623 0.36018 0.12146 0.55442 0.0000 
