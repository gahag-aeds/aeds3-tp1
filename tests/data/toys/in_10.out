0.217148 0.145449 0.192214 0.055203 0.053586 0.016489 0.039808 0.077428 0.124722 0.077954 